### PR TITLE
test: remove wait time to reduce indetermination

### PR DESCRIPTION
**Pull request by lance6716**
### What problem does this PR solve? 
part of https://github.com/pingcap/dm/issues/773

### What is changed and how it works?
remove wait time to reduce indetermination

### Check List 

Tests 

 - Integration test

Code changes

Side effects

Related changes

 - Need to cherry-pick to the release branch

### CODE REVIEW

File: syncer/streamer_controller.go [hunk 1]
```diff
@@ -223,24 +224,11 @@ func (c *StreamerController) GetEvent(tctx *tcontext.Context) (event *replicatio
...
-
-		t, err2 := time.Parse(time.UnixDate, str)
-		if err2 != nil {
-			tctx.L().Warn(fmt.Sprintf("cannot parse %s to time: %s", str, err2))
-		} else {
-			// mock upstream instance restart from 5-20s
-			startTime := t.Add(5 * time.Second)
-			endTime := startTime.Add(20 * time.Second)
-			now := time.Now()
-			if now.Before(endTime) && now.After(startTime) {
```
Comment (csuzhangxc) [inline syncer/streamer_controller.go:240]: we only fail in some times previously or only restart once is enough, @GMHDBJD this new implement can achieve your purpose?

Comment (GMHDBJD) [reply syncer/streamer_controller.go:240]: Enough

**Review by GMHDBJD — COMMENTED** [review]
LGTM

**Review by csuzhangxc — APPROVED** [review]
LGTM

**Pull request merged**